pkg/cmd/dependency_build.go

```diff
@@ -75,7 +76,8 @@ func newDependencyBuildCmd(out io.Writer) *cobra.Command {
 				man.Verify = downloader.VerifyIfPossible
 			}
 			err = man.Build()
-			if e, ok := err.(downloader.ErrRepoNotFound); ok {
+			var e downloader.ErrRepoNotFound
+			if errors.As(err, &e) {
```

Comment:
similar, and other `if errors.As(...)` usage

```suggestion
			if e := downloader.ErrRepoNotFound{}; errors.As(err, &e) {
```